java/java/src/main/java/org/whispersystems/libsignal/protocol/DecryptionErrorMessage.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (C) 2014-2016 Open Whisper Systems
+ *
+ * Licensed according to the LICENSE file in this repository.
+ */
+package org.whispersystems.libsignal.protocol;
+
+import org.signal.client.internal.Native;
+
+import org.whispersystems.libsignal.InvalidMessageException;
+import org.whispersystems.libsignal.ecc.ECPublicKey;
+import org.whispersystems.libsignal.util.guava.Optional;
+
+public final class DecryptionErrorMessage {
+
+  final long handle;
```

Comment:
`private`? as you have `public long nativeHandle() {` if needed.

Reply:
Oh, I think I meant for this to go the other way around: removing the `nativeHandle()`. If it turns out I still need it across packages I'll definitely make this `private`.